internal/service/batch/job_definition.go

```diff
@@ -327,6 +327,419 @@ func ResourceJobDefinition() *schema.Resource {
 				},
 			},
 
+			"ecs_properties": {
+				Type:          schema.TypeList,
+				MaxItems:      1,
+				Optional:      true,
+				ConflictsWith: []string{"container_properties", "node_properties", "eks_properties"},
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"task_properties": {
+							Type:     schema.TypeList,
+							MaxItems: 1,
+							Required: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"container": {
+										Type:     schema.TypeList,
+										Required: true,
```

Comment:
```suggestion
										Required: true,
										MaxItems: 10,
```

According to the docs: 
> You can include from one (1) to ten (10) containers in the EcsProperties structure. For more information see [EcsTaskProperties:containers](https://docs.aws.amazon.com/batch/latest/APIReference/API_EcsTaskProperties.html).
https://docs.aws.amazon.com/batch/latest/userguide/multi-container-jobs.html#multi-container-general